src/components/Footer/Footer.react.js

```diff
@@ -124,7 +124,7 @@ const Footer = () => {
                 <Link to="/devices">Devices</Link>
```

Comment:
every others are rendering with Link to why <a> for this one! any specific reasons for this change

Reply:
This page need to be reloaded as Link tag doesn't do so, it doesn't update the state.
So, either we need to use window.location.reload() or simply can use anchor tag.